sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -19,6 +19,13 @@ export class AzureKeyCredential implements KeyCredential {
     update(newKey: string): void;
 }
 
+// @public
+export class AzureSasCredential implements SasCredential {
```

Comment:
It feels odd to me that we use `Sas` and not `SAS`. It is an abbreviation, not a word in it self. Was this discussed in the API review meeting? 

cc @jsquire, @ellismg, @tg-msft

Reply:
The .NET language guidelines discourage consecutive capital letters, including in cases of abbreviation - so this would be the expected capitalization for the C# version.   I can't speak to language norms that we're using in JS/TS.... but I expect that this would follow whatever those are.

Reply:
Since we have `HttpClient` and such, there is some evidence that we've been willing to not preserve acronym casing

Reply:
Storage uses SAS so I kind of think that's what we should go with, but I agree that our codebase has examples of either. HTTPClient

Reply:
Change this to `AzureSASCredential` but kept the member named `signature`.  This is in line with other languages (they use `Sas` or `SAS` in the type name but still spell out `signature` for the member that holds the value instead of calling it `sas` or some casing thereof).